src/data/padics/padic_norm.lean

```diff
@@ -158,6 +159,17 @@ begin
     using padic_val_rat_def p n_nonzero,
 end
 
+lemma padic_val_nat_of_div_not_one {n p : nat} [prime : fact p.prime] (nonzero : n ≠ 0) (s : p ∣ n)
+  : 1 ≤ padic_val_nat p n :=
```

Comment:
Colon should go on the end of preceding line.
Also, the name of this lemma is confusing me.
I would call it `one_le_padic_val_nat_of_dvd`